src/sentry/tasks/scheduler.py

```diff
@@ -0,0 +1,33 @@
+
+from __future__ import absolute_import, division
+
+import logging
+
+from django.utils import timezone
+
+from sentry.app import locks
+from sentry.models import ScheduledJob
+from sentry.tasks.base import instrumented_task
+from sentry.utils.retries import TimedRetryPolicy
+
+logger = logging.getLogger('sentry')
+
+
+@instrumented_task(name='sentry.tasks.enqueue_scheduled_jobs')
+def enqueue_scheduled_jobs(**kwargs):
+    from sentry.celery import app
+
+    lock = locks.get('scheduler:process', duration=60)
+    with TimedRetryPolicy(5)(lock.acquire):
+        queryset = list(ScheduledJob.objects.filter(
```

Comment:
Ah, that's what I thought. I moved somethings around (as part of the warning when there's >100 jobs), so `queryset` should be appropriate now.